### DISCUSSION TITLE
Is it possible to exclude some sites from the cache

### DISCUSSION THREAD

**Discussion by YBS-PC** [AdguardTeam/AdGuardHome#3491] [Q&A]
Is it possible to exclude some sites from the dns cache? For example, I want all the responses to dns requests google.com they were received from the DNS server, and not from the AGH dns cache.

**Comment by ameshkov** [comment]
Nope, it's not possible

**Reply by xenstar** [reply]
Can you create an exclude section for optimistic cache? I am using a NAT VPS to set up my Proxy Server. When the server real IP is change, it will automatically update in real-time to cloudflare A record. But since Adguard Home cache the domain IP, I can't browse any website due to the cache. 

There should be an exclude section for optimistic cache, so that for a certain domain, it will always check the real domain IP instead of cache.

**Comment by donald2612** [comment]
Even for upstream-vitality testing it would be great for monitoring tools... very appreciated!
